maven-build-scan/action.yml

```diff
@@ -0,0 +1,71 @@
+name: Publish Maven Build Scan
+description: Publish Maven Build Scan
+
+inputs:
+  build-workflow-filename:
+    description: 'Filename of the workflow where the maven-build-scan/save action was triggered'
+    required: true
+  develocity-url:
+    description: 'Develocity URL'
+    required: true
+  develocity-access-key:
+    description: 'Develocity access key'
+    required: false
+  tos-location:
+    description: 'Terms of Service location as an URL (https://foo.com/tos.html) or a Github repository file (/<owner>/<repo>/blob/<branch>/tos.html)'
+    required: true
+  develocity-allow-untrusted:
+    description: 'Develocity allow-untrusted flag'
+    default: 'false'
+  pr-comment-tos-acceptance-missing:
+    description: 'pull-request comment added when Terms of Service are not accepted ({0} in the value will be replaced by tos-location input)'
+    default: 'Please accept [Develocity Terms of Service]({0}) to get your pull-request Build Scan published by commenting this pull-request with the following message:'
+  pr-comment-tos-acceptance-request:
+    description: 'pull-request comment to accept the Terms of Service'
+    default: 'I have read Develocity Terms of Service and I hereby accept the Terms'
+  pr-comment-tos-acceptance-validation:
+    description: 'pull-request comment added when Terms of Service are accepted'
+    default: 'All Contributors have accepted Develocity Terms of Service.'
+  signature-branch:
+    description: 'Git branch where the signature file will be stored'
+    default: ${{ github.event.repository.default_branch }}
+  signature-location:
+    description: 'Signature file location'
+    default: '.github/develocity-tos.json'
+  white-list:
+    description: 'CSV List of users not required to accept the Terms of Service'
+    default: ''
+  github-token:
+    description: 'The token used for Github API requests'
+    default: ${{ github.token }}
+    required: false
+
+runs:
+  using: composite
+  steps:
+    - name: Load data
+      id: load
+      uses: gradle/github-actions/maven-build-scan/load@v0.1
+      with:
+        build-workflow-filename: ${{ inputs.build-workflow-filename }}
+        pr-comment-tos-acceptance-request: ${{ inputs.pr-comment-tos-acceptance-request }}
+    - name: Verify Terms of Service acceptance
+      uses: gradle/github-actions/terms-of-service-acceptance/run@v0.1
```

Comment:
I am not super happy with the need for a fully qualified name of the composite action as the tag needs to be updated on each new release (which could easily be forgotten).

The alternative would be to checkout the `github-actions` code in the calling workflow to allow calling composite actions with relative paths, this looks worse. though

Reply:
The common solution to this is to use a version alias (like `@v1`) that always points to the latest release. This is done via a tag that is force pushed on each release.

If we do that, then this file won't need to be updated on each release, but it will still contain the fully-qualified path to the action (which I think is OK).

Reply:
Sounds good 👍